### ISSUE TITLE
Add package pandoc to skare3

### ISSUE THREAD

**Issue by mpr0v0st** [sot/skare3#352]
Add package pandoc to skare3

In an effort to make the Mission Planning Standard Ground Procedures more accessible and usable for the team, we are restructuring them. Time has been spent developing a python script to convert a procedure from a raw file to an online accessible, searchable, and linkable document. Further research reveals that the pandoc module offers exactly these features along with many others which have potential to serve the program over all in the future.

**Comment by taldcroft** [comment]
Some additional info:
- `pandoc` on conda is the binary install of pandoc, e.g. https://pandoc.org/installing.html.  This is not Python, just mostly something in bin.
-  pkgs/main seems to have an older version 2.2.3.2 vs. most recent 2.9.2.1 on conda-forge.
- There is also a `pypandoc` (version 1.5 on conda-forge) that is a thin Python wrapper.

tl;dr - use conda packages from conda-forge.

**Comment by javierggt** [comment]
would this go into a ska3-flight release?

**Comment by taldcroft** [comment]
Yes, along with the next ska3-matlab release.
